src/examples/avatar/AvatarShape.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { ReactComponent as ZendeskIcon } from '@zendeskgarden/svg-icons/src/26/zendesk.svg';
+import { ReactComponent as UserStrokeIcon } from '@zendeskgarden/svg-icons/src/16/user-solo-stroke.svg';
+
+const StyledCol = styled(Col)`
+  margin-bottom: ${p => p.theme.space.sm};
```

Comment:
Why is a margin being added to the bottom?

Reply:
If `margin` is needed for multiple rows, then it should be applied to the top of all `<Row>` components except for the first child. But in this case (and several that follow) there is only one row.

Reply:
I spoke with @rossmoody about the `margin-bottom`. I think he said it's not needed, so I removed it. Well have to remove it from the button examples too 👍.